src/utils.ts

```diff
@@ -121,7 +123,7 @@ const estimatePaymentGas = async ({
     // This may be wrong in case the feesReceiver performs some operations using
     // the msg.sender address in the fallback/receive function.
     return await estimateInternalCallGas({
-      from: feesReceiver,
+      from: tokenOrigin,
```

Comment:
If we introduce this back, we'll need to change the comments to clarify that it works on rskj, while in other EVMs may not work as expected (Anvil is an example)

Reply:
I decided to include a check against the chainID and set it to `tokenOrigin` in rskj and to `feesReceiver` in other cases